### ISSUE TITLE
Allow "request": "attach" in launch.json for a mix_task

### ISSUE THREAD

**Issue by andr-ec** [elixir-lsp/elixir-ls#1105]
Right now only "launch" is available.
I assume it would look something like this, is this possible? 

```
{
  "version": "0.2.0",
  "configurations": [
    {
      "name": "mix run",
      "type": "mix_task",
      "request": "attach",
      "startApps": false,
      "projectDir": "${workspaceFolder}"
    }
  ]
}
```

**Comment by lukaszsamson** [comment]
BTW this is a duplicate of https://github.com/elixir-lsp/elixir-ls/issues/834

**Comment by lukaszsamson** [comment]
Addressed in 77fcebb10469cc233d2a6a66a96602325b16d02b. Please head to README if you would like to check if it works in your case.
